.gitlab-ci.yml

```diff
@@ -777,6 +777,11 @@ library:ci-analysis:
 library:ci-paco:
   extends: .ci-template
 
+library:ci-itree:
+  extends: .ci-template
+  needs:
+    - library:ci-paco
```

Comment:
https://github.com/coq/coq/blob/master/dev/ci/README-users.md#add-your-project-by-submitting-a-pull-request
>If you only depend on Coq itself the implicit needs from the template suffices. Otherwise the needs list must include all transitive dependencies including build:base or build:edge+flambda depending on the switch you chose.